prow/config/jobs_test.go

```diff
@@ -40,6 +40,7 @@ func TestJobs(t *testing.T) {
 				return fmt.Errorf("private org must use private cluster, got %v", j.Base.Cluster)
 			}
 		case "istio":
+		case "istio-ecosystem":
```

Comment:
case doesn't fallthrough

```suggestion
		case "istio", "istio-ecosystem":
```

Reply:
too much C lately :/